moveit2.repos

```diff
@@ -1,14 +1,14 @@
 repositories:
   # Keep moveit_* repos here because they are released with moveit
+  geometric_shapes:
+    type: git
+    url: https://github.com/moveit/geometric_shapes.git
+    version: ros2
```

Comment:
Isn't geometric_shapes released already?

Reply:
yes, but on rolling it makes sense to always test against the latest version to detect upstream breakages immediately. The build farm already takes care of testing against binaries. All source install instructions suggest installing the source dependencies in the repos files as well. The scenario of building moveit with binaries only is not recommended or supported.

Reply:
Ok. Agreed.